evap/student/views.py

```diff
@@ -252,6 +252,8 @@ def render_vote_page(
     dropout: bool,
     for_rendering_in_modal: bool = False,
 ) -> HttpResponse:
+    if evaluation.main_language == evaluation.UNDECIDED_MAIN_LANGUAGE:
+        evaluation.main_language = "en"
```

Comment:
Hm, this is not quite right and makes me afraid that we display (or even save!) this wrongly somewhere later - a caller using `render_vote_page` likely does not expect that the evaluation gets changed in here.

What do you think about going a bit more defensive and defining an extra variable `fallback_language` which is `evaluation.main_language` or `"en"` if the main language is not decided. Then we use this variable in `request.GET.get("language", fallback_language)`. How do you feel about this change?